### ISSUE TITLE
Slide operation problem in Waymo, for boxes_3d and camera_parameters

### ISSUE THREAD

**Issue by thibo73800** [Visual-Behavior/aloception-oss#29]
For  `boxes_3d` and `camera_parameters` labels in `WaymoDataset`, it is impossible to perform the slider operation. The problem comes from the `SpatialTensorAugmented` module.

An example where the error is generated is the following:

```
import alodataset, aloscene

waymo = alodataset.WaymoDataset(
    split=Split.VAL,
    labels=["gt_boxes_3d"],
    sequence_size=2,
)

frames = next(iter(waymo.train_loader(batch_size = 8)))
frames = aloscene.Frame.batch_list([frame["front"] for frame in frames])

print(frames[:1,:1])

```

The error happens in the last line of code (frames[:1,:1]).

**Comment by ragier** [comment]
It seems to work now
